core/src/main/java/jenkins/cli/StopBuildsCommand.java

```diff
@@ -99,13 +99,13 @@ private void stopBuild(final Run build,
             try {
                 executor.doStop();
                 isAnyBuildStopped = true;
-                stdout.println(String.format("Build %s stopped for job %s;", buildName, jobName));
+                stdout.println(String.format("Build %s stopped for job %s", buildName, jobName));
             } catch (final Exception e) {
-                stdout.println(String.format("Exception occurred while trying to stop build %s for job %s;", buildName, jobName));
-                stdout.println(String.format("Exception class: %s, message: %s;", e.getClass().getSimpleName(), e.getMessage()));
+                stdout.println(String.format("Exception occurred while trying to stop build %s for job %s", buildName, jobName));
+                stdout.println(String.format("Exception class: %s, message: %s", e.getClass().getSimpleName(), e.getMessage()));
```

Comment:
This should be just one line, else it will be unclear that these are related.

Reply:
Agreed with @daniel-beck

Reply:
@oleg-nenashev It's already fixed in https://github.com/jenkinsci/jenkins/pull/3686/commits/6912362009f68b060973380568f409a3b9cee8f7